fdbserver/workloads/MachineAttrition.actor.cpp

```diff
@@ -132,17 +141,109 @@ struct MachineAttritionWorkload : TestWorkload {
 	virtual void getMetrics( vector<PerfMetric>& m ) {
 	}
 
-	struct UIDPredicate {
-		UIDPredicate(StringRef uid ) : uid( uid ) {}
-		bool operator() ( WorkerInterface rhs ) { return rhs.locality.zoneId() != uid; }
-	private:
-		StringRef uid;
-	};
+	static bool noSimIsViableKill(int coordFaultTolerance, int& killedCoord, std::vector<NetworkAddress> coordAddrs, WorkerDetails worker) {
```

Comment:
Generally in FoundationDB, we pass the parameters by pointer if they will be changed inside the function. Reference is used mostly as a const Reference. So it will be good to change killedCoord to pass by pointer. In that way it will be obvious by looking at the caller itself whether  an argument can be changed by the callee.